popup.html

```diff
@@ -1,7 +1,9 @@
 <!DOCTYPE html>
 <html>
 	<head>
-		<title>Dashboard</title>
+		<meta content="text/html;charset=utf-8" http-equiv="Content-Type">
+		<meta content="utf-8" http-equiv="encoding">
```

Comment:
where did you get http-equiv from?

Reply:
do you think any of these are unnecessary? I Googled an error about encoding not being specified and grabbed these meta tags.